### DISCUSSION TITLE
Welcome to frida-gadget Discussions!

### DISCUSSION THREAD

**Discussion by ksg97031** [ksg97031/frida-gadget#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by serg-yalosovetsky** [comment]
hi!
i trying to install modified file, and get error 
Performing Streamed Install
adb: failed to install .\diia.signed.apk: Failure [-124: Failed parse during installPackageLI: Targeting R+ (version 30 and above) requires the resources.arsc of installed APKs to be stored uncompressed and aligned on a 4-byte boundary]

i have samsung s23 ultra, with 14 android
apktool i install today, last version
can you help me?

**Reply by ksg97031** [reply]
Could you send the APK file to ksg97031@gmail.com?

**Comment by windayaniachmad** [comment]
Hi, @ksg97031 could you please give the example how to use option --force-manifest ? I cannot find the example of use case.
My decoded apk AndroidManifest.xml doesn't specify full package name it only shows  <activity android:exported="true" android:name=".presentation.splash.SplashscreenActivity" and it cause error when continue the process "The target class file /presentation/splash/SplashscreenActivity.smali was not found".

The smali is exist in folder com/packagename/presentation/splash/SplashscreenActivity.smali but since maybe in extracted AndroidManifest.xml doesn't specify the full path it becomes error.

I have used below script but still give the same result 
docker run -v $(pwd):/Users/laptop/ ksg97031/frida-gadget android.apk --arch arm64
docker run -v $(pwd):/Users/laptop/ ksg97031/frida-gadget android.apk --arch arm64 --use-aapt2
docker run -v $(pwd):/Users/laptop/ ksg97031/frida-gadget android.apk --arch arm64 --use-aapt2 --force-manifest

Many thanks
